ExtensionManifest.md

```diff
@@ -0,0 +1,88 @@
+# Extension Manifests - Ringtail 9.5.008
+Extension manifests are files that specify all the settings needed to setup and configure UI extensions. They are intended to be versioned assets that are updated in Ringtail with each new UI extension release.
+
+## Structure
+Extension manifests are authored in JSON format. Here's an example:
+```json
+{
+    "name": "Calculator",
+    "description": "A nifty 4-function calculator widget",
+    "type": "Workspace",
+    "url": "https://calculator.example.com",
+    "namePrefix": "CALC",
+    "fields": [{
+        "id": 1,
+        "name": "Sum",
+        "type": "Number",
+        "isOneToOne": true,
+        "codingSettings": "Write",
+        "excludeFromImport": true,
+    }],
+    "statistics": [{
+        "id": 1,
+        "name": "calculations performed"
+    }]
+}
+```
+
+## Top-Level Settings
+These settings define basic features required for every UIX.
+
+- `name` <[String]> Name of the UIX displayed in the case home tab list or documents workspace.
+- `type` <[String]> Location where the UIX will appear in Ringtail. Must be one of:
+  - `Workspace` - Documents area workspace pane
+  - `Case` - Case home page
+- `url` <[String]> URL of the web app to load.
+- `configuration` <[String]> (Optional) Payload stored by Ringtail and delivered to UIX upon initialization.
+- `description` <[String]> (Optional) Description to differentiate similar UIX displayed in the UIX admin area.
+- `privateKey` <[String]> (Optional) Key used to sign a JWT for authenticating Ringtail with backend UIX systems. See [security considerations](README.md#security-considerations) for more information.
+- `namePrefix` <[String]> (Optional) Prefix used to group and differentiate UIX fields and statistics from others in Ringtail. This value is required if fields or statistics are specified. **NOTE: This value cannot be changed once set.**
```

Comment:
By "others," do you mean other fields and statistics? Also, the value is required if UIX fields or statistics are specified, right? You might want to make these explicit, especially the second one. Pronouns can be confusing to keep track of.

Also, I'd change "once set" to "after it is set."

Reply:
Yes, this means to differentiate fields an statistics related to this UIX from other fields and statistics in Ringtail.